armi/reactor/components/complexShapes.py

```diff
@@ -91,6 +91,58 @@ def getCircleInnerDiameter(self, Tc=None, cold=False):
             return 0.0
 
 
+class HexHoledCircle(basicShapes.Circle):
+    """Circle with a uniform hexagonal hole hollowed out of it."""
```

Comment:
optional: ad the word "single" before uniform or hexagonal. its an important distinction between this and holled hex since holed hex can have multiple holes with nHoles